README.md

```diff
@@ -148,7 +148,10 @@ The Amazon Chime SDK for JavaScript uses WebRTC, the real-time communication API
 
 ## Installation
 
-Make sure you have Node.js version 12 or higher. Node 14 is recommended and supported.
+Make sure you have the following dependency installed:
+- Node.js version 12 or higher. Node 16 - the current LTS version - is recommended.
```

Comment:
- nit: `Node 16`  => `Node.js 16`
- How about removing `the current LTS version`?
  As of Jan 2022, v17 is **Current**, while v16 is **Active LTS**.

  Or you can specify and keep updating it regularly:
  `Node.js 16 (Active LTS) is recommended.`